postgis/docker-postgis
# alpine3.16 / pg15beta1 / postgis3.3.0alpha1 / improved README.md

Changes:
- Alpine upgrade to `alpine:3.16`
  - upstream changes : https://github.com/docker-library/postgres/pull/962
- improved `README.md` 
  - listing images and adding more information.
- Added new **test** images  ( Can be changed at any time, and does not block subsequent code refactorings ) 
  - `14-3.3.0alpha1-alpine`
  - `15beta1-3.3.0alpha1-alpine`
  - `15beta1-master`
- Refactored:  `./update.sh` script.
   -  added a placeholder Dockerfile for creating alpine only images
      -  IMHO: This is a nasty temporary workaround until the new `jq` based framework [is installed](https://github.com/postgis/docker-postgis/issues/299#issuecomment-1159120053) 
   -  improved logging
   -  generating markdown files for copy/pasting to the README.md
      -  `_dockerlists_debian.md`
      -  `_dockerlists_alpine.md`
      -  `_dockerlists_test.md`
- `latest` image tag now: `14-3.2`
- `*-master` ( test ) images has been updated
   -  `POSTGIS="3.3.0dev 3.3.0alpha1-94-g7c69026c8" [EXTENSION] PGSQL="140" GEOS="3.11.0beta3-CAPI-1.16.0" PROJ="9.1.0" LIBXML="2.9.10" LIBJSON="0.15" LIBPROTOBUF="1.3.3" WAGYU="0.5.0 (Internal)"`

Side comments:
* `15beta1-3.2.1`  is not working.

How to test:
*  check out this branch
* `make update`
   * check the temp files:  `_dockerlists_*.md`
